### PR TITLE
Integrated with latest swagger spec for management of policy certificates

### PR DESCRIPTION
**Pull request by gkostal**
**What**
---
 This is the update to the data-plane .NET SDK for the Azure Attestation Service based on the swagger changes made in this PR:

https://github.com/Azure/azure-rest-api-specs/pull/8298

**Tests**
---
New tests have been added, recordings have been generated and all tests are passing in playback mode.

### CODE REVIEW

File: eng/mgmt/mgmtmetadata/attestation_data-plane.txt [hunk 1]
```diff
@@ -3,12 +3,12 @@ AutoRest installed successfully.
 Commencing code generation
```
Comment (isra-fel) [inline eng/mgmt/mgmtmetadata/attestation_data-plane.txt]: Having this file under `mgmt/mgmtmetada/` doesn't seem right to me. @AlexGhiondea  Hi Alex, do you happen to know the excepted behavior for data plane?

**Comment by gkostal** [comment]
The test failures are not related to my update.  They are in the Microsoft.Azure.ServiceBus tests.
```
2020-02-27T00:28:38.2186697Z 00:28:38.2181644: Begin Microsoft.Azure.ServiceBus.UnitTests.AmqpConverterTests.Convert_SB_message_to_Amqp_message_and_back on .NETCoreApp,Version=v1.0
2020-02-27T00:28:38.2255995Z   X Microsoft.Azure.ServiceBus.UnitTests.API.ApiApprovals.ApproveAzureServiceBus [699ms]
2020-02-27T00:28:38.2256562Z   Error Message:
2020-02-27T00:28:38.2256823Z    Assert.Equal() Failure
2020-02-27T00:28:38.2257627Z                                   â†“ (pos 71843)
2020-02-27T00:28:38.2258462Z Expected: Â·Â·Â· { }\n        public override System.Threading.Tasks.Task<MicrÂ·Â·Â·
2020-02-27T00:28:38.2259239Z Actual:   Â·Â·Â· { }\n        public ManagedIdentityTokenProvider(Microsoft.AzÂ·Â·Â·
2020-02-27T00:28:38.2259915Z                                   â†‘ (pos 71843)
2020-02-27T00:28:38.2260232Z   Stack Trace:
2020-02-27T00:28:38.2260623Z      at ApprovalTests.Reporters.AssertReporter.AssertEqual(String approvedContent, String receivedContent)
2020-02-27T00:28:38.2261153Z    at ApprovalTests.Reporters.AssertReporter.AssertFileContents(String approved, String received)
2020-02-27T00:28:38.2261642Z    at ApprovalTests.Reporters.AssertReporter.Report(String approved, String received)
2020-02-27T00:28:38.2262104Z    at ApprovalTests.Reporters.FirstWorkingReporter.Report(String approved, String received)
2020-02-27T00:28:38.2262611Z    at ApprovalTests.Reporters.ContinousDeliveryUtils.ReportOnServer(String approved, String received)
2020-02-27T00:28:38.2263095Z    at ApprovalTests.Reporters.TfsVnextReporter.Report(String approved, String received)
2020-02-27T00:28:38.2263594Z    at ApprovalTests.Reporters.FirstWorkingReporter.Report(String approved, String received)
2020-02-27T00:28:38.2264066Z    at ApprovalTests.Approvers.FileApprover.ReportFailure(IApprovalFailureReporter reporter)
2020-02-27T00:28:38.2264570Z    at ApprovalTests.Core.Approver.Verify(IApprovalApprover approver, IApprovalFailureReporter reporter)
2020-02-27T00:28:38.2265079Z    at ApprovalTests.Approvals.Verify(IApprovalApprover approver, IApprovalFailureReporter reporter)
2020-02-27T00:28:38.2265623Z    at ApprovalTests.Approvals.Verify(IApprovalWriter writer, IApprovalNamer namer, IApprovalFailureReporter reporter)
2020-02-27T00:28:38.2266089Z    at ApprovalTests.Approvals.Verify(IApprovalWriter writer)
2020-02-27T00:28:38.2266465Z    at ApprovalTests.Approvals.Verify(String text, Func`2 scrubber)
2020-02-27T00:28:38.2267081Z    at Microsoft.Azure.ServiceBus.UnitTests.API.ApiApprovals.ApproveAzureServiceBus() in /home/vsts/work/1/s/sdk/servicebus/Microsoft.Azure.ServiceBus/tests/API/APIApprovals.cs:line 32
2020-02-27T00:28:38.2556623Z 00:28:38.2548821: End Microsoft.Azure.ServiceBus.UnitTests.AmqpConverterTests.Convert_SB_message_to_Amqp_message_and_back
```

**Comment by gkostal** [comment]
The test run for Microsoft.Azure.Attestation is succeeding:
```
2020-02-27T00:28:23.0311179Z Test run for D:\a\1\s\artifacts\bin\Microsoft.Azure.Attestation.Tests\Debug\netcoreapp2.1\Attestation.Tests.dll(.NETCoreApp,Version=v2.1)
2020-02-27T00:28:23.1085379Z Microsoft (R) Test Execution Command Line Tool Version 16.3.0
2020-02-27T00:28:23.1086586Z Copyright (c) Microsoft Corporation.  All rights reserved.
2020-02-27T00:28:23.1102752Z 
2020-02-27T00:28:23.2044335Z Starting test execution, please wait...
2020-02-27T00:28:23.2599911Z 
2020-02-27T00:28:23.2602483Z A total of 1 test files matched the specified pattern.
2020-02-27T00:28:25.6839776Z Results File: D:\a\1\s\sdk\attestation\Microsoft.Azure.Attestation\tests\TestResults\netcoreapp2.1.trx
2020-02-27T00:28:25.6841007Z 
2020-02-27T00:28:25.6841573Z Test Run Successful.
2020-02-27T00:28:25.6842097Z Total tests: 4
2020-02-27T00:28:25.6842629Z      Passed: 4
2020-02-27T00:28:25.6843801Z  Total time: 2.4127 Seconds
```

File: eng/mgmt/mgmtmetadata/attestation_data-plane.txt (hunk 1 above)
Comment (gkostal) [reply eng/mgmt/mgmtmetadata/attestation_data-plane.txt]: @isra-fel according to the docs, this file is expected.  [Link](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#sdk-generation-from-updated-spec)

>generate.ps1 will create a metadata file under eng\mgmt\mgmtmetadata\<rpName>.txt, have that file as part of your PR.

Comment (AlexGhiondea) [reply eng/mgmt/mgmtmetadata/attestation_data-plane.txt]: As this is an auto-generated project, I believe the file is expected.

**Review by AlexGhiondea — APPROVED** [review]

**Pull request merged**